perun-base/src/test/resources/perun-roles.yml

```diff
@@ -124,4 +124,56 @@ perun_policies:
       - PERUNOBSERVER:
     include_policies:
       - test_resource_admin
+
+  #SearcherEntry
+  getUsers_Map<String, String>_policy:
```

Comment:
Why is this policy different from the others? There is ", " between strings in the Map.

Reply:
Fixed